library/src/main/java/io/ona/kujaku/KujakuLibrary.java

```diff
@@ -0,0 +1,109 @@
+package io.ona.kujaku;
+
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.List;
+
+import io.ona.kujaku.activities.MapActivity;
+import io.ona.kujaku.callables.AsyncTaskCallable;
+import io.ona.kujaku.domain.Point;
+import io.ona.kujaku.listeners.OnFinishedListener;
+import io.ona.kujaku.tasks.GenericAsyncTask;
+import io.ona.kujaku.utils.Constants;
+
+import static io.ona.kujaku.utils.IOUtil.readInputStreamAsString;
+
+/**
+ * @author Vincent Karuri
+ */
+public class KujakuLibrary {
```

Comment:
Add tests for this Class. I can see you have written your own code here, what if someone else makes breaking changes since this is supposed to be the entry point into the MapActivity